dev-tools/validation/check-authz-route-matrix.py

```
@@ -61,64 +490,1977 @@ def string_list(value: Any, field: str, errors: list[str]) -> list[str]:
+                    f"token_profiles must not contain duplicate profile {profile!r}"
+                )
+            else:
+                profiles[profile] = values
+        if values.get("issuer") and values["issuer"] != TOKEN_ISSUER:
+            errors.append(f"{label}.issuer must be {TOKEN_ISSUER!r}")
+        if "kind" in raw_profile:
+            errors.append(f"{label} must use type instead of legacy kind")
+    return profiles
+
+
+def validate_role_assurance(document: dict[str, Any], errors: list[str]) -> set[str]:
```

Ruff (0.16.0)

[warning] Too many branches (23 > 12)

(PLR0912)